package.json

```diff
@@ -17,7 +17,7 @@
     "prepare": "husky install",
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
-    "cypress:headed": "cypress run --headed"
```

Comment:
Why was cypress removed?

Reply:
my bad, I must have removed this by mistake